src/common/components/Result/Result.module.scss

```diff
@@ -0,0 +1,70 @@
+.result {
+  margin: 10px;
+  list-style: none;
+  padding: 10px;
+  background: #fff;
+  position: relative;
+
+  &__name {
+    font-family: 'Nunito', sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: rgb(54, 54, 54);
+    cursor: pointer;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    max-height: 48px;
+
+    &:hover {
+      color: #17673d;
+      transition: color .3s ease,border-bottom .3s ease;
+    }
+  }
+
+  &__img {
+    background: #f8f7f2;
+    text-align: center;
+    overflow: hidden;
+    height: 0;
+    padding-top: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    position: relative;
+  }
+
+  &[data-pricedrop="true"] {
+    &::before {
+      content: "price drop";
+      position: absolute;
+      z-index: 2;
+      background-color: #db274b;
+      padding: 5px 10px 0 10px;
+      margin: 0;
+      color: #fff;
+      white-space: nowrap;
+      text-transform: uppercase;
+      font-family: 'Nunito', sans-serif;
+      font-weight: 700;
+      letter-spacing: 4px;
+      line-height: 16px;
+      left: calc(50%);
+      transform: translateX(-50%);
+    }
```

Comment:
So there are multiple approaches to toggling classes. Conventionally folks will take advantage of props in a component to toggle styles, e.g. in `31` of `src/common/components/Result/Result.js` you can do a check and apply a new className to it:

```
  <li className={pricedrop ? `${styles.result} ${styles.result__pricedrop}` : styles.result} />
```